app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -26,6 +26,15 @@
             {{#t}}Download Firefox for mobile devices.{{/t}}
           </p>
 
+          <form class="marketing-sms-link">
```

Comment:
I could talk myself into several places.

* [marketing-email-client->sendSms](https://github.com/mozilla/fxa-content-server/pull/3573/files#diff-384d29d76ca063c3ff14b4d386cd7becR69). Whether the number should/shouldn't contain dashes, dots, spaces, etc seems a bit like a detail that mostly concerns the server. Another upside is the code is shared if we ever decide to add the form somewhere else. The downside is it's far from where the input originates.
* [marketing-email-snippet-ios->submit](https://github.com/mozilla/fxa-content-server/pull/3573/files#diff-09a4038d36a8de3fc0edfaa32e736e6bR91) because that's close to the user input. The downside is the code wouldn't be shared if we move the form elsewhere.
* [marketing-email-prefs->sendSms](https://github.com/mozilla/fxa-content-server/pull/3573/files#diff-0a6b3e960373826cb3909c011bbe6757R128). Shared code, seems reasonable, but far from user input.